x-pack/plugins/fleet/server/types/models/package_policy.ts

```diff
@@ -36,6 +36,7 @@ const PackagePolicyBaseSchema = {
   enabled: schema.boolean(),
   package: schema.maybe(
     schema.object({
+      id: schema.string(),
```

Comment:
should we have a `schema.maybe()` for retro compatibility ?

Reply:
Yes good call.